packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -756,9 +761,24 @@ class TextSelectionGestureDetector extends StatefulWidget {
   /// time (within [kDoubleTapTimeout]) to a previous short tap.
   final GestureTapDownCallback onDoubleTapDown;
 
+  /// Called after [onDoubleTapDown] when the pointer is lifted, unless
+  /// [onDoubleLongTapStart] is triggered.
+  final GestureTapUpCallback onDoubleTapUp;
+
   /// Called when a mouse starts dragging to select text.
   final GestureDragStartCallback onDragSelectionStart;
 
+  /// Called after [onDoubleTapDown], when the second tap is sustained for longer
+  /// than [kLongPressTimeout], but not necessarily lifted.
+  final GestureLongPressStartCallback onDoubleLongTapStart;
+
+  /// Called after [onDoubleLongTapStart], when the pointer is dragged.
+  final GestureLongPressMoveUpdateCallback onDoubleLongTapMoveUpdate;
```

Comment:
Is this not going to be called if you double tapped and dragged right away?

Reply:
I don't think so,  the the second touch down (or long press + drag) gesture is recognized via the long tap gesture recognizer, the raw gesture recognizer doesn't seem to have a suitable recognizer for recognizing touch down and drag:

https://github.com/flutter/flutter/blob/6b4719009a070909b4d1fa508470827f4e6aa92b/packages/flutter/lib/src/widgets/text_selection.dart#L965-L1029